common/src/main/java/bisq/common/taskrunner/Task.java

```diff
@@ -65,7 +67,7 @@ protected void failed(String message) {
 
     protected void failed(Throwable t) {
         log.error(errorMessage, t);
-        taskHandler.handleErrorMessage(errorMessage);
+        taskHandler.handleErrorMessage(format("%s: %s", errorMessage, t.getMessage()));
```

Comment:
The UI might get a bit verbose on errors with this change.

Reply:
I reverted it, then pushed a change (631c3f4) -> `taskHandler.handleErrorMessage(t.getMessage());`, which is what I meant to do in the first place.  Now, the verbose msg in logged on the server, but just the exception msg is passed up to clients.